static/src/javascripts/components/raven-js/raven.js

```diff
@@ -1410,7 +1410,14 @@ var Raven = {
      */
     lastEventId: function() {
         return lastEventId;
-    }
+    },
+
+    /*
+     * Has raven been configured?
+     *
+     * @return {boolean}
+     */
+    isSetup: isSetup
```

Comment:
if it's compatible, then you could always take the new `raven-js` version, and codify the dependency. The rationale is fine, it just might mislead a developer who is wondering how we are accessing new api on an older module!